exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/QaServiceImplIntegrationTest.java

```diff
@@ -308,6 +312,30 @@ void getBlockTransactions() {
     }
   }
 
+  @Test
+  @RequiresNativeLibrary
+  void afterCommit() throws CloseFailuresException {
+    try (MemoryDb db = MemoryDb.newInstance();
+        Cleaner cleaner = new Cleaner()) {
+      Fork view = db.createFork(cleaner);
+      setServiceNode(node);
+      service.initialize(view);
+
+      BlockCommittedEvent event = mock(BlockCommittedEvent.class);
+      service.afterCommit(event);
+
+      // Check that the afterCommit counter was updated.
```

Comment:
This test does not seem correct — after commit cannot synchronously make any changes to the database.